diff -ruN ezmlm-web-2.1.orig/htaccess.sample ezmlm-web-2.1/htaccess.sample
--- ezmlm-web-2.1.orig/htaccess.sample	Mon Sep 25 20:49:46 2000
+++ ezmlm-web-2.1/htaccess.sample	Sat Dec 30 18:41:13 2000
@@ -3,7 +3,7 @@
 #order deny,allow
 #deny from all
 #allow from .ru.ac.za
-AuthName "EZ Mailing List Manager
+AuthName 'EZ Mailing List Manager'
 AuthType Basic
-AuthUserFile /etc/ezmlm/.htusers
+AuthUserFile AUTHFILE
 require valid-user
diff -ruN ezmlm-web-2.1.orig/index.c ezmlm-web-2.1/index.c
--- ezmlm-web-2.1.orig/index.c	Mon Sep 25 20:14:18 2000
+++ ezmlm-web-2.1/index.c	Sat Dec 30 18:45:00 2000
@@ -8,7 +8,7 @@
 
 int main(void) {
    /* Change this path to wherever you decided to put ezmlm-web.cgi */
-   execv("/usr/local/bin/ezmlm-web.cgi"); 
+   execv("/home/httpd/html/ezmlm/ezmlm-web.cgi"); 
 
    /* Note that you could also use the following to allow a specific user
       to store their mailing lists in a different directory defined by
